frame/liquidations/src/mock/runtime.rs

```diff
@@ -189,6 +190,26 @@ impl WeightToFeePolynomial for WeightToFee {
 	}
 }
 
+pub struct XcmFake;
+impl Into<Result<cumulus_pallet_xcm::Origin, XcmFake>> for XcmFake {
+	fn into(self) -> Result<cumulus_pallet_xcm::Origin, XcmFake> {
+		todo!("please test via local-integration-tests")
+	}
+}
+impl From<Origin> for XcmFake {
+	fn from(_: Origin) -> Self {
+		todo!("please test via local-integration-tests")
+	}
+}
+impl SendXcm for XcmFake {
+	fn send_xcm(
+		_destination: impl Into<xcm::latest::MultiLocation>,
+		_message: xcm::latest::Xcm<()>,
+	) -> xcm::latest::SendResult {
+		todo!("please test via local-integration-tests")
+	}
+}
+
```

Comment:
This is duplicated in lending & dutch auction, is there any way to export it from one of the crates? or perhaps have an 'angular-mocks-common' crate (or something along those lines)

Reply:
i will think on optimization in next PR.